app/tasks.py

```diff
@@ -58,6 +58,26 @@ def addItem(donation_f, description_f, particulars_f, manufacturer_f, model_f, q
         item_bulk.append(Item(tax_receipt_no=donation_f, description=description_f, particulars=particulars_f, manufacturer=manufacturer_f, model=model_f,
                               quantity=quantity_f, working=working_f, condition=condition_f, quality=quality_f, batch=batch_f, value=value_f, verified=True))
 
+    def getTeleExt(telephone):
+        if "xxxxxx" not in telephone:
+            teleNo = telephone.partition(" ext.")[0]
```

Comment:
Can you check the different formatting in the CSV and tell me the number formatting? I think 10b csv is formatted by CSV. We probably want to actually support both 10b csv and the Weebly CSV in the long term.